sandcastle/api.py

```diff
@@ -381,32 +344,29 @@ def run(self, command: Optional[List] = None):
         logger.info("Sandbox pod is deployed.")
         return self.get_logs()
 
-    def exec(self, command: List[str]) -> str:
-        """
-        exec a command in a running pod; if any mapped dirs are set,
-        sync the dirs before and after the execution
-
-        :param command: command to run
-        :returns logs
-        """
-        # https://github.com/kubernetes-client/python/blob/master/examples/exec.py
-        for m_dir in self.mapped_dirs:
-            self.copy_path_to_pod(m_dir)
-        # FIXME: we're unable to get RC of the exec
-        ws_client: WSClient = stream(
+    def _do_exec(self, command: List[str], preload_content=True):
```

Comment:
return type `WSClient` is missing in method definition

Reply:
because WSClient is not the only possible value to be returned; but I can add that